### PR TITLE
Improvements to naming

### PR DESCRIPTION
**Pull request by klasmodin**

### CODE REVIEW

File: lie_group_diffeo/action.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ def inf_action(self, lie_grp_element):
         """
         raise NotImplementedError('abstract method')
 
-    def inf_action_adj(self, v, m):
+    def momentum_map(self, v, m):
         """Return the infinitessimal action, an `lie_group.associated_algebra`
```
Comment (adler-j) [inline lie_group_diffeo/action.py]: We need to update the doc-strings as well.

**Review by adler-j — CHANGES_REQUESTED** [review]
One minor comment, fix and then we can merge.

**Comment by klasmodin** [comment]
OK, updated some doc-strings, and a few new occurrences of momentum_map.

**Comment by adler-j** [comment]
Great! Merging.

**Pull request merged**